lesson10.md

```diff
@@ -0,0 +1,234 @@
+<!-- .slide: id="lesson10" -->
+
+# Basic Frontend - Fall 2021
+
+Lesson 10, Tuesday, 2021-10-19
+
+---
+
+### Lesson overview
+
+* recap
+* introducing JavaScript objects
+
+---
+
+### Recap: types
+
+What Javascript types have we seen so far?
+
+* Number
+<!-- .element: class="fragment" -->
+* String
+<!-- .element: class="fragment" -->
+* Boolean
+<!-- .element: class="fragment" -->
+* Undefined
+<!-- .element: class="fragment" -->
+
+---
+
+### Recap: types
+
+```js
+let age = 39;
+let firstName = "Carlo";
+let lastName = "Trimarchi";
+let nationality = "Italian";
+let cityOfBirth = "Messina";
+let favoriteColor = "green";
+let hairColor = "black";
+let canSpeakGerman = false;
+let isVegetarian = true;
+let hadDinner = hour > 20; // false
```

Comment:
maybe just remove this last one which could create some confusion

Reply:
Done :)